src/authorizations/components/redesigned/CustomeApiTokenOverlay.test.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
```

Comment:
can you rename this file to `CustomApiTokenOverlay.test.tsx` (lose the `e` after `Custom`)